### PR TITLE
challah-bread

### PR DESCRIPTION
**Pull request by siggines**
A kind of braided bread preperation.

### CODE REVIEW

**Comment by siggines** [comment]
First check failed as I put a comma after tag by mistake.

File: src/challah-bread.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Challah Bread
+
+A kind of braided bread preperation.
```
Comment (hallzy) [inline src/challah-bread.md:3]:
```suggestion
A kind of braided bread preparation.
```

File: src/challah-bread.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+1. Fill a large bowl with the water, and sprinkle with the yeast.
+2. Whisk together: honey, oil, 2 eggs, and salt. Add to bowl.
+3. Add flour to bowl in small amounts, beating/whisking after each addition.
+4. Once the mixtures becomes a viscous dough, begin kneading until smooth and elastic, and no longer sticky.
+5. Cover bowl with a damp cloth and let rise for 1-2 hours, or until dough has doubled in size.
+6. Sprinkle flour onto a chopping board, place the dough onto it and divide in half.
+7. Knead each half, adding flour as needed to keep the dough non-sticky.
+8. Divide each half into thirds and roll into a long snake about 4cm (1.5in) in girth.
+9. Pinch the ends of the each three snakes together firmly and plait them together, making two items of braided dough.
+10. Grease two baking trays or use baking paper, cover with dry cloth and let rise for one hour. Preheat oven to 190 C / Gas mark 5.
```
Comment (hallzy) [inline src/challah-bread.md:32]:
```suggestion
10. Grease two baking trays or use baking paper, cover with dry cloth and let rise for one hour. Preheat oven to 190°C / Gas mark 5.
```

File: src/challah-bread.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+11. Whisk an egg and brush onto each plait, sprinkle with seeds.
+12. Bake until bread has a hollow sound when tapped, cool on a rack for an hour before slicing.
+
+## Contribution
+
+http://jacobsiggins.co.uk
```
Comment (hallzy) [inline src/challah-bread.md:38]:
```suggestion
- siggines - [website](http://jacobsiggins.co.uk)
```

**Review by hallzy — CHANGES_REQUESTED** [review]

**Comment by siggines** [comment]
^ Thanks

**Review by hallzy — APPROVED** [review]

**Review by DGKSK8LIFE — APPROVED** [review]

**Pull request merged**